packages/snaps-cli/src/commands/build/build.e2e.test.ts

```diff
@@ -21,20 +18,19 @@ describe('mm-snap build', () => {
         expect.stringMatching(/Checking the input file\./u),
       );
       expect(runner.stdout).toContainEqual(
-        expect.stringMatching(/Building the snap bundle\./u),
+        expect.stringMatching(/Building the Snap bundle\./u),
       );
-      expect(runner.stderr).toContainEqual(
-        expect.stringMatching(
-          /Compiled \d+ files? in \d+ms with \d+ warnings?\./u,
-        ),
+
+      expect(runner.stdout).toContainEqual(
+        expect.stringMatching(/Compiled \d+ files? in \d+ms\./u),
```

Comment:
Since manifest warnings don't show up here anymore would this string only contain build errors/warnings now?